pages/common/john.md

```diff
@@ -0,0 +1,36 @@
+# john
+
+> Password cracker.
+> More information: <https://www.openwall.com/john/>.
+
+- Crack password hashes:
+
+`john {{path/to/hashes.txt}}`
+
+- Show passwords cracked:
+
+`john --show {{path/to/hashes.txt}}`
+
+- Display user's cracked password by user identifier from multiple files:
+
+`john --show --users={{0}} {{path/to/hashes*}} {{path/to/other/hashes*}}`
+
+- Crack password hashes, using a custom wordlist:
+
+`john --wordlist={{path/to/wordlist.txt}} {{path/to/hashes.txt}}`
+
+- Crack password hashes, using a specific hash format:
+
+`john --format={{md5crypt|...}} {{path/to/hashes.txt}}`
+
+- List available hash formats:
```

Comment:
Perhaps it would make more logical sense if this example was before the `Crack password hashes, using a specific hash format` example?

Reply:
OK, I inverted the examples.